requirements.txt

```diff
@@ -12,4 +12,5 @@ colour
 flask-cors
 pep8
 nose
-Flask-Migrate
\ No newline at end of file
+Flask-Migrate
+Flask-Login
```

Comment:
We should use flask-scrypt, add that as a dep. It has almost the same interface as flask-bcrypt, which it looks like you are using